hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -137,6 +136,14 @@
       .withDocumentation("There are cases when extra files are requested to be deleted from metadata table which was never added before. This config"
           + "determines how to handle such spurious deletes");
 
+  public static final ConfigProperty<Boolean> RECORDKEY_DE_DUPLICATE = ConfigProperty
+      .key("_" + METADATA_PREFIX + ".recordkey.deduplicate")
```

Comment:
lets try to maintain same naming across the board. 
"_" + METADATA_PREFIX + "exclude.key.from.payload"

Reply:
fixed